OpenDreamClient/Interface/Controls/ScalingViewport.cs

```diff
@@ -292,6 +292,14 @@ private void EnsureViewportCreated()
 
             DebugTools.AssertNotNull(_viewport);
         }
+
+        Matrix3 IViewportControl.GetWorldToScreenMatrix() {
```

Comment:
The `ScalingViewport` control is pretty much lifted straight from SS14. You can probably just steal [their implementation](https://github.com/space-wizards/space-station-14/blob/80cfba535a821638f61f0ea6f4892e83a80a7cb5/Content.Client/Viewport/ScalingViewport.cs#L272) of these as well.